api/routes/projects.js

```diff
@@ -85,6 +86,28 @@ router.get('/', async (req, res, next) => {
     }
     return;
   }
+
+  function linkData(ResArray, ItArray) {
+    const FinalArray = [];
+    for (var i = 0; i < ResArray.length; i++) {
```

Comment:
To be consistent with the `const`/`let` way of using variables, you could change the `var` in the first two of these `for` loops to `let` and, change `var` to `const` in the last two (the `for...in` loops)